docs/api-center-analyzer-integration-nodejs.md

```diff
@@ -0,0 +1,134 @@
+<!-- markdownlint-disable MD033 -->
+# API Center Analyzer Integration
+
+You can analyze your API documents using Visual Studio Code (Standalone) or Azure Portal (Server-Side). This section will guide you through the integration process.
+
+## Standalone Analyzer through Visual Studio Code
+
+> To use this standalone analyzer, you need to install the [API Center extension](https://marketplace.visualstudio.com/items?itemName=apidev.azure-api-center) in Visual Studio Code.
+
+1. Install dependencies.
+
+     ```javascript
+    cd nodejs/webapi && npm install
+    ```
+
+1. Run the sample app.
+
+    ```javascript
+    npm start
+    ```
+
+   You'll see the following Swagger UI page.
+
+   ![Swagger UI - weather forecast](./images/api-center-analyzer-integration-nodejs-01.png)
+
+1. Navigate to http://localhost:3030/api-docs/swagger.json to see the OpenAPI document.
+
+   ![OpenAPI - weather forecast](./images/api-center-analyzer-integration-nodejs-02.png)
+
+1. Save the OpenAPI document to `weatherforecast.json`.
+1. Stop the sample app by pressing <kbd>Ctrl</kbd>+<kbd>C</kbd>.
+1. Open `weatherforecast.json` in Visual Studio Code. You'll see many yellow lines and red lines
+
+   ![OpenAPI - standalone analysis](./images/api-center-analyzer-integration-nodejs-03.png)
+
+1. Type <kbd>F1</kbd> then select "Azure API Center: Set active API Style Guide".
+
+   ![Standalone API Analysis](./images/api-center-analyzer-integration-04.png)
+
+   You will have several options to choose the styles. Choose either `Microsoft Azure REST API` or `OWASP API Security Top 10` and see the analysis results.
+
+   ![Standalone API Analysis - Choose style](./images/api-center-analyzer-integration-05.png)
+
+1. You can also choose the custom API style guide that reflects your organization's policies. This time choose `Select Local File` then select the `resources/rulesets/oas.yaml` file. You'll still see many yellow lines and red lines. Hover your mouse and see what needs to be fixed.
+
+   ![Standalone API Analysis - analysis result](./images/api-center-analyzer-integration-06.png)
```

Comment:
This screenshot has `"version": "1.0"`, but your screenshot has `1.0.0`. Please update your `swagger/openapiservice.js` code to align with this screenshot.